addons/test_mail/tests/test_message_track.py

```diff
@@ -130,6 +142,33 @@ def test_message_track_template_at_create(self):
         # one email send due to template
         self.assertSentEmail(self.record.env.user.partner_id, [self.partner_admin], body='<p>Hello Test</p>')
 
+    def test_message_track_template_at_create_from_message(self):
+        """Make sure records created through aliasing show the original message before the template"""
+        # setup
+        test_model = self.env['ir.model']._get('mail.test.ticket')
+        custom_values = {'name': 'Test', 'customer_id': self.user_admin.partner_id.id,
+                         'mail_template': self.env.ref('test_mail.mail_test_ticket_tracking_tpl').id}
+        self.env['mail.alias'].create({
+            'alias_name': 'groups',
+            'alias_user_id': False,
+            'alias_model_id': test_model.id,
+            'alias_contact': 'everyone',
+            'alias_defaults': custom_values})
+        record = self.format_and_process(MAIL_TEMPLATE, '"Sylvie Lelitre" <test.sylvie.lelitre@agrolait.com>',
+                                         'groups@test.com', target_field='customer_id', subject=custom_values['customer_id'],
+                                         target_model='mail.test.ticket')
+        self.flush_tracking()
+
+        # Should be trigger message and response template
+        self.assertEqual(len(record.message_ids), 2)
+        messages = list(record.message_ids)
+        messages.sort(key=lambda msg: msg.id)
+        trigger = messages[0]
+        template = messages[1]
+        self.assertIn('Please call me as soon as possible this afternoon!', trigger.body)
+        self.assertIn('Hello', template.body)
```

Comment:
We could probably check the complete rendered body (as it is not that long) ? Just because "hello" is a bit short and not crytal clear as a comparison :D